lib/app/smoge_app.dart

```diff
@@ -1,10 +1,13 @@
 import 'package:flutter/material.dart';
 import 'package:smoge/app/app_theme.dart';
+import 'package:smoge/generated/l10n.dart';
 import 'package:smoge/ui/tab_bar/navigation_container.dart';
 
 class SmogeApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) => MaterialApp(
+        localizationsDelegates: [S.delegate],
```

Comment:
proponuję zmienić nazwę tej klasy z `S` na `Strings`. Można to zrobić w pubspecu jako paramter `class_name`